src/DotVVM.Framework/Resources/Scripts/DotVVM.ts

```diff
@@ -555,7 +555,7 @@ class DotVVM {
         return new Promise<() => Promise<DotvvmAfterPostBackEventArgs>>(async (resolve, reject) => {
             const viewModelName = options.viewModelName!;
             const viewModel = this.viewModels[viewModelName].viewModel;
-            
+
```

Comment:
@exyi I'll leave these two last reformatting changes in. I'm fine with trimming whitespace-only lines automatically.

Reply:
Thats fine, I'm not likely to get in the history of this line :)